server/admin_management/api/app/routers/router_access_control_privilege.py

```diff
@@ -76,35 +76,32 @@ def create_access_control_privilege_many(
     )
 
     try:
-        audit_event_log.requesting_user = user_service.get_user_by_cognito_user_id(
-            requester.cognito_user_id
-        )
+        audit_event_log.requesting_user = requester
         audit_event_log.role = role_service.get_role_by_id(
             access_control_privilege_request.role_id
         )
         audit_event_log.application = audit_event_log.role.application
 
-        return access_control_privilege_service.create_access_control_privilege_many(
+        new_delegated_admin = access_control_privilege_service.create_access_control_privilege_many(
```

Comment:
I feel maybe just use "response" as variable... this service method returns a list of response objects.

Reply:
yeah thanks Ian I'll change it